app/views/profile/index.html.slim

```diff
@@ -43,7 +45,12 @@
         .sm-col.sm-col-5.px1 = t('.phone')
         .sm-col.sm-col-7.px1 = @decrypted_pii.phone
 
-h2.h3.my0 = t('headings.profile.login_info')
+h2.h3.my0
+  = t('headings.profile.login_info')
+  - if current_user.decorate.identity_not_verified?
```

Comment:
same comment as above -- can we move logic into decorator class and out of the view?